src/Backups/registerBackupEngineAzureBlobStorage.cpp

```diff
@@ -124,6 +124,8 @@ void registerBackupEngineAzureBlobStorage(BackupFactory & factory)
         }
 
 
+        params.use_same_password_for_base_backup = false;
```

Comment:
This line looks strange, why is it here?

Reply:
Thanks for catching, of course it should be `use_same_s3_credentials_for_base_backup`. Fixed.